src/main/groovy/org/ajoberstar/gradle/git/release/semver/NearestVersionLocator.groovy

```diff
@@ -37,87 +38,97 @@ import org.slf4j.LoggerFactory
  * @since 0.8.0
  */
 class NearestVersionLocator {
```

Comment:
Based on the diff, it looks like the whitespace was changed in this file. Mind changing it back to tabs for consistency with the rest of the project?